src/lib/y2partitioner/actions/controllers/clone_partition_table.rb

```diff
@@ -71,7 +71,8 @@ def suitable_devices_for_cloning?
         #
         # @return [Array<Y2Storage::Partitionable>]
         def suitable_devices_for_cloning
-          @suitable_devices ||= working_graph.blk_devices.select { |d| suitable_for_cloning?(d) }
+          @suitable_devices_for_cloning ||= working_graph.blk_devices
```

Comment:
In general, in this commit, why are all those renames needed?

Reply:
it is new cop, that check for consistent caching of method call. This cop check that cache is kept in variable that has same name as method which result is cached. ( I hope I describe it clear ).

Reply:
and link to cop https://rubocop.readthedocs.io/en/stable/cops_naming/#namingmemoizedinstancevariablename

Reply:
Ok, now I got it. Thanks!